### PR TITLE
Clarify the role of implicit conversion language feature imports

### PR DESCRIPTION
**Pull request by janekdb**

### CODE REVIEW

**Comment by janekdb** [comment]
Review by @SethTisue

File: tutorials/tour/implicit-conversions.md [hunk 1]
```diff
@@ -48,4 +48,11 @@ implicit def int2Integer(x: Int) =
...
 
-To define your own implicit conversions, you must first `import scala.language.implicitConversions` (or invoke the compiler with `-language:implicitConversions`). The feature must be explicitly enabled because it has pitfalls if used indiscriminately.
+Because implicit conversions can have pitfalls if used indiscriminately the compiler warns when compiling the implicit conversion definition.
+
+To turn off the warnings take either of these actions:
+
+* Import `scala.language.implicitConversions` into the scope of the implicit conversion definition
+* Invoke the compiler with `-language:implicitConversions`
+
+No warning is emitted when the conversion is applied by the compiler.
```
Comment (janekdb) [inline tutorials/tour/implicit-conversions.md]: `is used by the compiler` clearer?

Comment (SethTisue) [inline tutorials/tour/implicit-conversions.md]: seems fine to me as is

**Comment by SethTisue** [comment]
excellent, thank you Janek!

**Pull request merged**